app/angular/src/server/config/webpack.config.js

```diff
@@ -67,9 +67,13 @@ export default function(configDir) {
           loaders: [require.resolve('ts-loader'), require.resolve('angular2-template-loader')],
         },
         {
-          test: /\.(html|css)$/,
+          test: /\.html$/,
           loader: 'raw-loader',
-          exclude: /\.async\.(html|css)$/,
+          exclude: /\.async\.css$/,
```

Comment:
Why do we need to exclude a css file is it doesn't pass the `test` anymore?

Reply:
oops, I think I need to exclude html here.